### ISSUE TITLE
Cannot add AES to the Accessibility list of Apps in macOS 14.4. 

### ISSUE THREAD

**Issue by carlca** [LiveEnhancementSuite/LESforMacOS#48]
**Describe the bug**
I cannot add AES to the Accessibility list of Apps in macOS 14.4. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Startup AES
2. Get the following warning...
![SCR-20240309-mvyh](https://github.com/LiveEnhancementSuite/LESforMacOS/assets/260612/a3da0f77-9eef-4733-95c4-15d0af7154ce)
3. Go to `System Settings` -> `Privacy and Security` -> `Accessibility`
4. Note that LES is not listed, and press the + button to add an app.
5. Navigate to `/Applications/Live Enhancement Suite`, select it and press OK.

**Expected behaviour**
I expected LES to be added to the list of apps in the `Accessibility` list.

**Actual behaviour**
LES was not added to the `Accessibility` list which meant that it could not be enabled, thus preventing further installation.

**Desktop (please complete the following information):**
 - macOS 14.4 Sonoma
 - Mac Mini M1 - 16GB

**Additional context**

**Comment by taomuller** [comment]
for me it appears in the accessibility tab right away but when i launch it it says: 

Live Enhancement Suite is only validated to run between macOS 11 and macOS 13 and is currently being run on macOS 14.4.
The program may behave in an undefined manner and may cause disruption but will continue running until prompted to exit.
If you believe this is in error or that the program must be updated to support a newer release of macOS, please file an issue at https://github.com/LiveEnhancementSuite/LESforMacOS/issues.

then hammerspoon opens and gives me the following error:

*** ERROR: ...ite.app/Contents/Resources/extensions/hs/les/LESmain.lua:258: attempt to concatenate a nil value (local 'subfoldername')
stack traceback:
	...ite.app/Contents/Resources/extensions/hs/les/LESmain.lua:258: in function 'buildPluginMenu'
	...ite.app/Contents/Resources/extensions/hs/les/LESmain.lua:632: in function 'reloadLES'
	...ite.app/Contents/Resources/extensions/hs/les/LESmain.lua:664: in main chunk
	[C]: in function 'dofile'
	/Users/xxx/.les/init.lua:16: in main chunk
	[C]: in function 'xpcall'
	...uite.app/Contents/Resources/extensions/hs/_coresetup.lua:698: in function 'hs._coresetup.setup'
	(...tail calls...)

LES doesnt show up in the control centre and does not work in ableton. i am also on macOS 14.4 Sonoma, macbook air m1

**Comment by taomuller** [comment]
i just got it to work! first of all, delete the current LES and empty your bin. then open Finder - Go to Folder - type ~/.les, hit enter and delete all files in that folder. Restart your mac. Now download v1.3.4.beta.3 and install. Right click - open and it should work.
